doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -108,6 +108,16 @@ Note: with an underlying IaaS this is possible, but then it introduces (undesira
 
 > **Gap description:** Allocation of hugepages from the same NUMA node as other resources of a Pod. To support this [cAdvisor needed a change to support NUMA](https://github.com/google/cadvisor/pull/2304). Changes in Kubernetes are planned to be implemented in the [Node Topology Manager](https://github.com/kubernetes/enhancements/issues/693).
 
+<a name="6.2.9"></a>
+### 6.2.0 User namespaces in Kubernetes
+
+> **Related requirements:** `e.man.004`, `sec.gen.006`
```

Comment:
Why sec.gen.006 - The Platform must support Secure logging. Logging with root account must be prohibited when root privileges are not required.

Suggest sec.sys.007 -- see below for definition

I think instead of just listing the requirement number we should provide the requirement for readability.

```suggestion
> **Related requirements:**
| Reference | Requirement |
|---|---|
| e.man.004 | Capability to isolate resources between tenants |
| sec.sys.007 | The Platform must implement controls enforcing separation of duties and privileges, least privilege use and least common mechanism (Role-Based Access Control). |
```